src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheck.java

```diff
@@ -180,6 +185,7 @@ public void leaveToken(DetailAST ast) {
             case TokenTypes.INSTANCE_INIT:
             case TokenTypes.STATIC_INIT:
             case TokenTypes.COMPACT_CTOR_DEF:
+            case TokenTypes.LAMBDA:
                 leaveMemberDef(ast);
```

Comment:
I also do not like namings for `leaveMemberDef` and `visitMemberDef`, since we visit not only `METHOD_DEF`, it is little confusing, we can discuss appropriate names here

maybe `visitEnclosingScope` or `visitContainerNode` or smth else, what do you think?

Reply:
``` visitContainerNode ``` sounds elegant